lib/swagger-module.ts

```diff
@@ -65,14 +65,19 @@ export class SwaggerModule {
   ) {
     const httpAdapter = app.getHttpAdapter();
     const finalPath = validatePath(path);
-    const swaggerUi = loadPackage('swagger-ui-express', 'SwaggerModule', () =>
-      require('swagger-ui-express')
+    const {
+      jsonSpecPath = finalPath + '-json',
+      swaggerUiLib = 'swagger-ui-express', 
+      ...swaggerUiOptions
+    } = options;
+    const swaggerUi = loadPackage(swaggerUiLib, 'SwaggerModule', () =>
+      require(swaggerUiLib)
     );
```

Comment:
This won't work with webpack. The following code:

```typescript
const swaggerUi = loadPackage('swagger-ui-express', 'SwaggerModule', () =>
      require('swagger-ui-express')
```
must be preserved, somehow.

Reply:
I am not very familiar with webpack, does it scan for require('swagger-ui-express')?
If so, would making `swaggerUiLib ? require(requireswaggerUiLib) : require('swagger-ui-express')` work?

Reply:
> I am not very familiar with webpack, does it scan for require('swagger-ui-express')?

Correct.

> If so, would making swaggerUiLib ? require(requireswaggerUiLib) : require('swagger-ui-express') work?

Instead of passing the library name as the `swaggerUiLib`, I'd suggest supporting the following syntax: 

```typescript
swaggerUiLib: require('your-swagger-ui-lib')
```

Reply:
Added SwaggerUiExpress interface instead of lib name